graph.go

```diff
@@ -201,7 +201,7 @@ func relPath(base, path string) string {
 	}
 
 	// TODO(sqs): hack
-	if strings.HasPrefix(rp, "../../../") && dockerCWD != "" {
+	if strings.HasPrefix(rp, filepath.Join("..", "..", ".."+string(filepath.Separator))) && dockerCWD != "" {
```

Comment:
`filepath.Join("..", "..", ".."+string(filepath.Separator))` is a constant and the `string(filepath.Separator)` at the end will get cleaned out by `filepath.Join`.

You should change it to `filepath.Join("..", "..", "..") + string(filepath.Separator)`.

```bash
~ $ goe 'filepath.Join("..", "..", ".."+string(filepath.Separator))'
(string)("../../..")
~ $ goe 'filepath.Join("..", "..", "..") + string(filepath.Separator)'
(string)("../../../")
```